api_gateway/test/variables.tf

```diff
@@ -0,0 +1,78 @@
+variable "user_tags" {
+  description = "Mandatory tags for the api-gateway resources"
+  type        = map(string)
+  default     = {}
+}
+
+variable "octopus_tags" {
+  description = "Octopus Tags"
+  type        = map(string)
+  default     = {}
+}
+
+variable "name" {
+  description = "The name of the API Gateway that has to be created"
+  type        = string
+  default     = null
+}
+
+variable "description" {
+  description = "The description of the API Gateway"
+  type        = string
+  default     = null
+}
+
+variable "aws_region" {
```

Comment:
Remove this variable. aws_region will be set as env variable (AWS_REGION) in octopus